webserver/views/api.py

```diff
@@ -77,14 +78,15 @@ def submit_listen():
 @api_bp.route("/1/user/<user_id>/listens")
 def get_listens(user_id):
 
+    listenstore = listenstore_connection.get_listenstore()
```

Comment:
Maybe it's better to move that closer to the place where it's used (line 89)? Or just get rid of variable completely?